sdks/python/setup.py

```diff
@@ -140,7 +140,7 @@ def get_version():
     # oauth2client >=4 only works with google-apitools>=0.5.18.
     'google-apitools>=0.5.18,<=0.5.20',
     'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
-    'googledatastore==7.0.1',
+    'googledatastore==7.0.1; python_version < "3.5.0"',
```

Comment:
Let's change version qualifier to `python_version < "3.0"`, since AFAIK googledatastore will not work on any version of Python 3. We can also add a `#TODO(BEAM-...)` comment, in case anyone reading this line will wonder.
Note that we used `python_version < "3.5.0"` on `typing` is a back-port module required only for Python interpreter less that 3.5.

Reply:
Done! Thanks for the feedback.